benches/banking_stage.rs

```diff
@@ -145,10 +152,16 @@ fn bench_banking_stage_multi_accounts(bencher: &mut Bencher) {
                 })
                 .collect();
 
-        let verified_setup_len = verified_setup.len();
+        let _verified_setup_len = verified_setup.len();
```

Comment:
Why add an underscore and not delete these?

Reply:
I'll remove the line (it was there before my PR)

Reply:
Actually, just leave the line unmodified. I'm moving the benches back to libtest right now, so I'll delete them in my PR. one less rebase!

Reply:
sounds good.